File: cfgs/pythoncfgs/DIVISIBILITY_9_USING_BITWISE_OPERATORS.py
Nodes
1;conditional;foo_1 == 0 or foo_2 == 9;8;
2;conditional;foo_1 < 9;10;
3;statement;return fun_3(fun_2(foo_1 >> 3) - fun_3(foo_2 & 7));12;
4;statement;return false;11;
5;statement;return true;9;
6;exit;;
Edges
0;2;5;
1;3;4;
2;6;
3;6;
4;6;
5;

File: cfgs/java_cfgs_foo/DIVISIBILITY_9_USING_BITWISE_OPERATORS.java
Nodes
1;special;;;
2;conditional;(foo_1 == 0) || (foo_2 == 9);14;
3;statement;return true;14;
4;exit;;;
5;conditional;foo_1 < 9;15;
6;statement;return false;15;
7;statement;return fun_1(((int) (foo_1 >> 3)) - ((int) (foo_2 & 7)));16;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;7;
5;4;
6;4;
